catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/NitfAttributeImpl.java

```diff
@@ -67,6 +70,26 @@ protected NitfAttributeImpl(
     this.shortName = shortName;
     this.longName = attributeName;
     this.accessorFunction = accessorFunction;
+    this.extAccessorFunction = accessorFunction;
+    this.attributeDescriptors = new HashSet<>();
+    this.attributeDescriptors.add(attributeDescriptor);
+    if (StringUtils.isNotEmpty(extNitfName)) {
+      this.attributeDescriptors.add(
+          ExtNitfUtility.createDuplicateDescriptorAndRename(extNitfName, attributeDescriptor));
+    }
+  }
+
+  protected NitfAttributeImpl(
```

Comment:
could we have this constructor call the other one and just add the `extAccessorFunction`?

Reply:
Fixed.